samples/benchmarks/coremark/prj.conf

```diff
@@ -0,0 +1,40 @@
+# Common coremark configuration
+CONFIG_COREMARK=y
+
+CONFIG_COREMARK_ITERATIONS=2000
+
+# Single thread execution
+CONFIG_COREMARK_THREADS_NUMBER=1
+# Choose memory method
+CONFIG_COREMARK_MEMORY_METHOD_STACK=y
+#CONFIG_COREMARK_MEMORY_METHOD_MALLOC=y
+
+# Supported only for single thread. No stack or heap required
+#CONFIG_COREMARK_MEMORY_METHOD_STATIC=y
+
+# Choose proper memory size based on memory method selected above
+# For 1 thread
+CONFIG_MAIN_STACK_SIZE=4096
+#CONFIG_HEAP_MEM_POOL_SIZE=4096
+
+# For 4 threads
+CONFIG_TIMESLICE_SIZE=10
+#CONFIG_MAIN_STACK_SIZE=10240
+#CONFIG_HEAP_MEM_POOL_SIZE=10240
```

Comment:
Consider (low prio for me) maybe creating separate overlay files to provide predefined configs without the need to modify prj.conf. 
Or maybe consider adding some predefined configurations in the sample.yaml?

Reply:
Sometime ago Vova decided config to look like that. I also dislike this approach but do not think it is worthy spending time modifying it. If you insist I will do it. If not lets leave it as it is.

Reply:
I'll not insist, IMO we can keep it like that for now.

Reply:
It's very unintuitive to have an alternative configuration behind the comment. I think we should use overlay file or another prj configuration for the CoreMark variant with threads. It is unclear what needs to be changed to run it with 4 threads

Reply:
If we do it in separate conf files we have problem with remote images which would not add extra overlay file by default.

Reply:
I see, thanks for pointing this out.

Maybe snippets would help in that case:

https://github.com/nrfconnect/sdk-nrf/pull/14315#discussion_r1522949793